example-locale/README.md

```diff
@@ -0,0 +1,186 @@
+
+## Prerequisites
+
+- A Recent Version of NodeJS
+- NPM (Feel free to use Yarn if you want to, just switch out the NPM Commands for their YARN counterparts)
+- [A locally running appwrite instance](https://appwrite.io/docs/installation).
+
+## Getting Started
+
+To get started quickly we will use [Create React App](https://github.com/facebook/create-react-app) to create the boilerplate that our code will be built on.
+
+```shell
+npx create-react-app appwrite-react
+cd appwrite-react
+```
+
+While we are in the CLI we will also install Appwrite by running:
+
+```shell
+npm install appwrite
+```
+
+and finally we will launch the React development server with:
+
+```shell
+npm start
+```
+
+This should launch a server on `localhost:3000` with Live Reload.
+
+## Introducing the Appwrite
+
+With the boilerplate now complete, we can now initialise Appwrite in the project before working on the details page. To keep things clean we will initialise this in it's own file, we will create a folder in `src/` and call it `config` and within this folder we will create the file `index.js`. Within this file go ahead and paste the following code:
+
+```js
+const Appwrite = require("appwrite"); //Import appwrite
+
+let sdk = new Appwrite();
+sdk
+  .setEndpoint("http://localhost:443/v1") // Your API Endpoint
+  .setProject("[PROJECT ID]") // Your project ID
+
+let response = sdk.locale.get();
+
+export {response};
+```
+
+A deeper inspection of this code can be found in the comments within it,
+
+TL:DR: Create a appwrite instance and initalise it with the endpoint and ProjectID of the project we are working with then export this for usage outside of the file.
+
+## Creating App.js
+
+Inside our `src/App.js` we'll be importing our two components `components/Header/js` and `components/YourInfo.js`. Header.js is just our logo and heading, where as YourInfo.js accepts the reponse of the Locale-API and displays it in the DOM.
+
+Make a components folder in the src folder and inside it add two files, Header.js and YourInfo.js.
+For now paste the following code in the `src/App.js`.
+
+```js
+import React, {Fragment} from "react";
+import "./App.css";
+import YourInfo from "./components/YourInfo";
+import Header from "./components/Header";
+
+function App() {
+  return (
+    <Fragment>
+      <div className="App">
+        <Header />
+        <YourInfo />
+      </div>
+    </Fragment>
+  );
+}
+
+export default App;
+```
+
+Inside our App.js we have just imported our two components. Now it's time to create YourInfo.js
+
+## Creating YourInfo.js
+
+This file recieves all the json data from the Local API. We first import the response which is basically `sdk.get();` from our index.js file. Then we use a promise to fetch the json data.
+
+We are using functional component in this file and we'll flood our state inside this promise only.
```

Comment:
Do you think `flood our state` is the right terminology?

Reply:
Ya, maybe we can change this to, "set our state inside this promise".